src/utils/interactiveHandlers.ts

```diff
@@ -0,0 +1,24 @@
+export default [
+  "click",
+  "contextmenu",
+  "dblclick",
+  "doubleclick",
+  "drag",
+  "dragend",
+  "dragenter",
+  "dragexit",
+  "dragleave",
+  "dragover",
+  "dragstart",
+  "drop",
+  "keydown",
+  "keypress",
+  "keyup",
+  "mousedown",
+  "mouseenter",
+  "mouseleave",
+  "mousemove",
+  "mouseout",
+  "mouseover",
+  "mouseup"
+]
```

Comment:
in the project there is a `.json` file with some static data, since this feel more like static data in my opinion, I'd like best to see this in the form of a `.json` file to match the `htmlElements.json` that already exist in this `utils` folder.

There is an exemple of it's use on https://github.com/vue-a11y/eslint-plugin-vuejs-accessibility/blob/42b2917b7a696777da1edb1a6051937f67871e11/src/rules/click-events-have-key-events.ts#L4

Reply:
Updated